nodes/srl/srl.go

```diff
@@ -80,6 +80,9 @@ var (
 		"ixrh3":    "7220IXRH3.yml",
 		"ixrh4":    "7220IXRH4.yml",
 		"ixrh432d": "7220IXRH4-32D.yml",
+		"ixrh5":    "7220IXRH5-32D.yml",
+		"ixrh564d": "7220IXRH5-64D.yml",
+		"ixrh564o": "7220IXRH5-64O.yml",
```

Comment:
thanks @sacckth 

I think there is also https://github.com/srl-labs/containerlab/blob/main/schemas/clab.schema.json that needs to feature those as legitimate types